src/Suave/Utils/Aether.fs

```diff
@@ -0,0 +1,223 @@
+﻿module Suave.Utils.Aether
```

Comment:
Yes, that would be nice. We could lift it in using paket and then change the module name using https://github.com/haf/YoLo/blob/master/README. @kolektiv: are you fine with this?